docs/generate-tables.sh

```diff
@@ -1,31 +1,30 @@
 #!/bin/bash
 # Generate the documentation tables as docs/generated/OJP.html from the .xsd schema files
 #
-# You need the binary `xsltproc`
+# You need the binary `java`
 # apt-get install xsltproc
 
 # The -e flag causes the script to exit as soon as one command returns a non-zero exit code
 set -e
 
-echo "Generating documentation tables ..."
-
-base_dir="$(dirname "${0}")/.."
+base_dir="$(readlink -f $(dirname "$0")/..)"
 xsl_dir=$base_dir/docs
 generated_dir="${base_dir}/docs/generated"
+basex="/tmp/basex.jar"
 
-# prepare generated_dir
-mkdir -p "${generated_dir}"
-rm -f "${generated_dir}"/OJP-prep.xml "${generated_dir}"/*.adoc "${generated_dir}"/*.html
-cp "${xsl_dir}"/asciidoc.css "${generated_dir}"/
+echo "Download BaseX ..."
```

Comment:
"Download BaseX" should only be echoed if actually downloading, thus inside of the wget-if